app-emulation/vagrant/vagrant-2.2.10.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+# Drop ruby27: Many dependencies do not support it
+# USE_RUBY="ruby25 ruby26 ruby27"
+USE_RUBY="ruby25 ruby26"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="vagrant.gemspec"
+RUBY_FAKEGEM_EXTRAINSTALL="keys plugins templates version.txt"
+RUBY_FAKEGEM_TASK_DOC=""
+
+inherit bash-completion-r1 ruby-fakegem
+
+DESCRIPTION="A tool for building and distributing development environments"
+HOMEPAGE="https://vagrantup.com/"
+SRC_URI="https://github.com/hashicorp/vagrant/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+virtualbox"
```

Comment:
This can be moved to `optfeature` since it doesn't have any function in this ebuild.

Reply:
I (as ex vargant maintainer) think it should stay.
it's possible to use vagrant with other providers via plugins, but those are separate, not packaged, sometimes paid. 95% people use it with virtualbox.